myPackage/Player.java

```diff
@@ -1,14 +1,27 @@
 package myPackage;
 
+
 public class Player {
+    private int playerId;
     private String name;
     private boolean isBatsman;
     private boolean isBowler;
+    private boolean isWicketKeeper;
+    private int teamId;
+    private String teamName;
 
```

Comment:
Do we need isCaptain well?

Reply:
Can we have created a Player interface? Bowler and batsmen could have been their implementation?